docs/hub/model-release-checklist.md

```diff
@@ -88,6 +91,8 @@ Bonus: a recognised library also allows you to track downloads of your model ove
    
    Try this model directly in your browser: [Space Demo](https://huggingface.co/spaces/username/model-demo)
    ```
+   
+It's a good practise to load the model from the model's own repository in the demo instead of loading them from drive or keeping the checkpoint in the repository. This way, Space is linked to the repository, and chances of your model making it to trending is higher due to increased downloads.
```

Comment:
```suggestion
When you write a demo, please download the model from its repository in the Hub (instead of using external sources) so both are cross-linked together. This will help all your artifacts achieve greater visibility.
```